ui/page/main_page.go

```diff
@@ -505,13 +542,22 @@ func (mp *MainPage) LayoutTopBar(gtx layout.Context) layout.Dimensions {
 											})
 									}),
 									layout.Rigid(func(gtx C) D {
-										return layout.Center.Layout(gtx, func(gtx C) D {
-											return components.LayoutBalance(gtx, mp.Load, mp.totalBalance.String())
-										})
+										return mp.totalDCRBalance(gtx)
 									}),
 									layout.Rigid(func(gtx C) D {
 										return mp.LayoutUSDBalance(gtx)
 									}),
+									layout.Rigid(func(gtx C) D {
+										mp.hideBalance.Icon = mp.Icons.RevealIcon
+										if mp.isBalanceHidden {
+											mp.hideBalance.Icon = mp.Icons.ConcealIcon
+										}
+										return layout.Inset{Top: values.MarginPadding1}.Layout(gtx, func(gtx C) D {
+											return layout.E.Layout(gtx, mp.hideBalance.Layout)
+										})
+
+									}),
+									
```

Comment:
The spacing between the icon and the balance doesn't seem inline with the mockup, add a left margin as needed

<img width="363" alt="Screenshot 2021-10-16 at 6 24 10 PM" src="https://user-images.githubusercontent.com/25265396/137596607-7b5877cb-10c6-46a4-9d70-af36437130db.png">